maxim/tests/test_fireworks.py

```
@@ -0,0 +1,495 @@
+
+dotenv.load_dotenv()
+
+fireworksApiKey = os.getenv("FIREWORKS_API_KEY")
+apiKey = os.getenv("MAXIM_API_KEY")
+baseUrl = os.getenv("MAXIM_BASE_URL") or "https://app.getmaxim.ai"
+repoId = os.getenv("MAXIM_LOG_REPO_ID")
+
+class TestFireworks(unittest.TestCase):
+    """Test class for Fireworks AI integration with Maxim logging."""
+
+    def setUp(self):
```

Ruff (0.11.9)

Missing return type annotation for public function `setUp`

Add return type annotation: `None`

(ANN201)